HHS/Head-Start-TTADP
# [TTAHUB-2418] Add language to AR and TR

## Description of change

Added language select to both AR and TR (session). This allows the user to select either 'English', 'Spanish', or both.

The review and complete forms have also been updated to show these values.

## How to test

- Create an AR with languages make sure you can update the values and hook up with with form validation.
- Verify AR review and approved pages.
- Create a TR with the session language make sure you can update the values and hook up with with form validation.
- Verify the view event page.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2418


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
